crates/proto/src/version/v662/enums/puv.rs

```diff
@@ -0,0 +1,460 @@
+pub mod Puv {
```

Comment:
Seems like quite a lot of module nesting... why dont you just call the enum `LegacyLevelSoundEvent`.. adjust the file name and remove the other mods?